docs/contributor/updateability.md

```diff
@@ -0,0 +1,650 @@
+# Upgrading MicroShift
+
+Following document is meant to be a contributor's introduction
+to the feature of MicroShift's updateability.
+
+## On-disk artifacts
+
+New on-disk artifacts were added:
+- `/var/lib/microshift-backups/`
+    - `health.json`
+    - backup directories
+- `/var/lib/microshift/version`
+
+### `/var/lib/microshift-backups/`
+
+New, additional directory for keeping health and backup related artifacts.
+For ostree systems it's a non-configurable directory where backups are created.
+For non-ostree systems it's a default (configurable) directory for backups.
+
+#### `health.json` [ostree-only]
+
+Contains information about health of the system.
+It's updated by green and red script supplied by MicroShift.
+
+Because MicroShift's healthcheck takes some time to assess the state of the MicroShift,
+this file might be updated after couple minutes into system boot.
```

Comment:
Let's add a command here to display the status of greenboot, so that the user knows when the file is updated.